ckanext/ontario_theme/templates/snippets/search_form.html

```diff
@@ -0,0 +1,41 @@
+{% ckan_extends %}
```

Comment:
It wasn't clear which filters/keywords are active, or how to eliminate the keyword. I wanted the user to be able to see all of their active searches in one place. It also helps clarify the filters on the left, because there might be only one type, but not because it is filtered on that type, but just because it is the only option left...

Yes, the "and" was partially for accessibility but also partially to be clear.

I do think they improve usability.